gslib/gcs_json_api.py

```diff
@@ -223,7 +223,13 @@ def __init__(self, bucket_storage_uri_class, logger, status_queue,
 
     self.global_params = apitools_messages.StandardQueryParameters(
         trace='token:%s' % trace_token) if trace_token else None
+
     additional_http_headers = {}
+
+    gs_json_host_header = config.get('Credentials', 'gs_json_host_header', None)
```

Comment:
I was going to suggest also adding this for `gs_host`, but it turns out that Boto already supports this:
https://github.com/gsutil-mirrors/boto/blob/91ba037e54ef521c379263b0ac769c66182527d7/boto/provider.py#L215

So no action needed. Mostly just leaving this comment here as a reminder to my future self :)